docs/topics/execute.adoc

```diff
@@ -68,49 +69,49 @@ $ <MTA_HOME>/bin/mta-cli -Dwindup.decompiler=procyon --input <INPUT_ARCHIVE_OR_D
 ----
 
 [discrete]
-[id='cli-bash-completion_{context}']
-== About {ProductShortName} {CLINameTitle} bash completion
+[id="cli-bash-completion_{context}"]
+== About {ProductShortName} {CLINameTitle} Bash completion
 
-The {ProductShortName} {CLIName} provides an option to enable bash completion for Linux systems, allowing the {ProductShortName} command-line arguments to be auto completed by pressing the Tab key when entering the commands. For instance, when bash completion is enabled, entering the following displays a list of available arguments.
+The {ProductShortName} {CLIName} provides an option to enable Bash completion for Linux systems, allowing the {ProductShortName} command-line arguments to be auto completed by pressing the Tab key when entering the commands. For instance, when Bash completion is enabled, entering the following displays a list of available arguments.
 
 [source,options="nowrap",subs="+quotes"]
 ----
 $ <MTA_HOME>/bin/mta-cli [TAB]
 ----
 
 [discrete]
-[id='bash-completion-temporary_{context}']
-=== Enabling bash completion
+[id="bash-completion-temporary_{context}"]
+=== Enabling Bash completion
 
-To enable bash completion for the current shell, execute the following command. After the prompt returns, follow the steps for using the {CLIName}.
+To enable Bash completion for the current shell, execute the following command. After the prompt returns, follow the steps for using the {CLIName}.
 
 [source,options="nowrap",subs="+quotes"]
 ----
 $ source <MTA_HOME>/bash-completion/mta-cli
 ----
 
 [discrete]
-[id='bash-completion-persistent_{context}']
-=== Enabling persistent bash completion
+[id="bash-completion-persistent_{context}"]
+=== Enabling persistent Bash completion
 
-The following commands allow bash completion to persist across restarts:
+The following commands allow Bash completion to persist across restarts:
 
-* To enable bash completion for a specific user across system restarts, include the following line in that user's `~/.bashrc` file.
+* To enable Bash completion for a specific user across system restarts, include the following line in that user's `~/.bashrc` file.
 +
 [source,options="nowrap",subs="+quotes"]
 ----
 source <MTA_HOME>/bash-completion/mta-cli
 ----
 
-* To enable bash completion for all users across system restarts, copy the {ProductName} {CLIName} bash completion file to the `/etc/bash_completion.d/` directory. By default, this directory is only writable by the root user.
+* To enable Bash completion for all users across system restarts, copy the {ProductName} {CLIName} Bash completion file to the `/etc/bash_completion.d/` directory. By default, this directory is only writable by the root user.
```

Comment:
"copy the {ProductName} {CLIName} Bash completion file to the `/etc/bash_completion.d/` directory. By default, this directory is only writable by the root user."

Change to "copy the {ProductName} {CLIName} Bash completion file to the `/etc/bash_completion.d/` directory as the root user."